docs/android.md

```diff
@@ -1,66 +1,40 @@
-- Go back to Build Settings window
-  - Select option “Export Project” 
-    <br><img src="images/android/exportProject.png" width='400px'>
-  - Export UnityProject to androidBuild folder, the folder structure should look like this
-    <br><img src="images/android/exportedProjectFolder.png">
-    
-**3. Add Unity Library module to NativeAndroidApp**
-<br>Do the following to add the exported androidBuild/unityLibrary module to the NativeAndroidApp Gradle project in Android Studio:
-- Open NativeAndroidApp in Android Studio
-- Open settings.gradle file
-  - Add a new project pointing to unityLibrary module after the main app module
-    ```
-    include ':unityLibrary'
-    project(':unityLibrary').projectDir=new File('..\\UnityProject\\androidBuild\\unityLibrary')
-    ```
-  - And add the following in dependencyResolutionManagement{repositories{ block
-    ```
-    flatDir {
-      dirs "${project(':unityLibrary').projectDir}/libs"
-    }
-    ```
-    <img src="images/android/settingsGradle.png">
-- Open build.gradle(Module: NativeAndroidApp.app) file
-  - Add the following in dependencies{ block
-    ```
-    implementation project(':unityLibrary')
-    implementation fileTree(dir: project(':unityLibrary').getProjectDir().toString() + ('\\libs'), include: ['*.jar'])
-    ```
-    <img src="images/android/buildGradleApp.png">
-  
-  - In the same file take a look at android{defaultConfig{ndk{ block and make sure abiFilters match the architectures you selected in Unity editor before exporting the project. The filter must match architectures in Unity editor exactly. If Unity exports only ARMv7 architecture, but the filter includes arm64-v8a, the application will crash on ARM64 devices. Check for valid abiFilters values in the [official Android documentation](https://developer.android.com/ndk/guides/abis#sa).
-  <br><img src="images/android/buildGradleAppAbiFilters.png">
-
-- Copy the contents of the gradle.properties file from the exported Unity project root folder to the gradle.properties file in the native application root folder. Note: if you update the Unity project and re-export it again, make sure that the contents of the gradle.properties file in the exported project did not change. If they did - repeat this step.
-  <br><img src="images/android/exportedASProject.png"><img src="images/android/gradlePropertiesApp.png">
-- Click Sync Now to do a project sync since Gradle files have been modified
-  <img src="images/android/syncGradle.png">
-- If everything succeeds, you should see unityLibrary module added in Android view
-  <img src="images/android/unityLibraryModule.png">
+- Go back to Build Profiles window.
+  - Select option “Export Project”, and Export UnityProject to a folder. (If you see Multiple application entries pop-up, click Yes.) 
+    <br><img src="images/android/exportProject.png" width='670px'>
 
 ## Project is ready
 Everything is ready to build, run and debug:
```

Comment:
I think buildOnDevice.png screenshot is broken, I mean does the main app is called 'app' in Android Studio ?

Reply:
Oh! I missed it. I will update the image as well. Thank you.